sw/device/lib/dif/dif_uart.h

```diff
@@ -10,6 +10,8 @@
 
 #include "sw/device/lib/base/mmio.h"
 
+#define DIF_UART_FIFO_SIZE_BYTES 32u
```

Comment:
Do you think you could replace this with a `const`? if not, can you at least add a Doxygen comment?

Reply:
Good point, this is not used anywhere in the header and can be a const. I will add a comment as well.

Reply:
Done.